### PR TITLE
Mobile nézet 2

### PR DESCRIPTION
**Pull request by kurtvonbleicher**
lsz approv

### CODE REVIEW

**Review by takacsniki — DISMISSED** [review]

File: assets/css/main.scss [hunk 1]
```diff
@@ -393,6 +394,11 @@ body {
 }
 
 
+.mobile_menu {
+    display: none;
+}
+
```
Comment (sapesz) [inline assets/css/main.scss:398]: pls remove unnecessary newlind

File: assets/css/main.scss [hunk 2]
```diff
@@ -439,4 +445,57 @@ body {
...
\ No newline at end of file
+    .vintage_container {
+        display: inline;
+        .vintage_img {
+            width:-webkit-fill-available;
+        }
+    }
+
+    .item_box:hover {
+
```
Comment (sapesz) [inline assets/css/main.scss:455]: pls remove newline

File: assets/css/main.scss [hunk 3]
```diff
@@ -439,4 +445,57 @@ body {
...
+    /*mobilnézet menü*/
+
+    .mobile_menu {
+        width: 35px;
+        height: 27px;
+        display: block;
+
+        .mobile_dropdown_container {
+            display: none;
+        }
```
Comment (sapesz) [inline assets/css/main.scss]: pls add newline

File: assets/css/main.scss [hunk 4]
```diff
@@ -439,4 +445,57 @@ body {
...
+        .mobile_menu_element {
+            width: 35px;
+            height: 5px;
+            background-color: snow;
+            margin: 6px 0;
+        }
```
Comment (sapesz) [inline assets/css/main.scss:472]: pls add newline

File: assets/css/main.scss [hunk 5]
```diff
@@ -439,4 +445,57 @@ body {
...
+        &:hover {
+
```
Comment (sapesz) [inline assets/css/main.scss:475]: pls remove newline

File: assets/css/main.scss [hunk 6]
```diff
@@ -439,4 +445,57 @@ body {
...
+                position: relative;
+                z-index: 10;
+                margin: none;
+                text-align: left;
+
+                .mobile_dropdown_item {
+                    width: 200px;
+                    justify-content: flex-start;
+                    background-color: snow;
+                    list-style: none;
```
Comment (sapesz) [inline assets/css/main.scss:489]: pls add newline

File: guitars_acoustic.html [hunk 7]
```diff
@@ -38,7 +38,7 @@ <h2>Akusztikus Gitárok</h2>
     <p>Ezek itt a menő akusztikus gitárok</p>
     <br>
     <main class="main">
-        <a class="item_link" href="">
+        <a class="item_link" href="index.html">
```
Comment (sapesz) [inline guitars_acoustic.html]: “/“ as href should be enough

File: index.html [hunk 8]
```diff
@@ -41,6 +41,36 @@
...
         <p>Az innováció a jelszavunk, az Ati gitárok a családunk</p>
         <br>
+
+        <div class="mobile_menu">
+            <div class="mobile_menu_element"></div>
+            <div class="mobile_menu_element"></div>
+            <div class="mobile_menu_element"></div>
+            <ul class="mobile_dropdown_container">
+                <li class="mobile_dropdown_item">
+                    <a href="index.html">Kezdőlap</a>
```
Comment (sapesz) [inline index.html]: “/“ should be enough

**Review by takacsniki — APPROVED** [review]

**Pull request closed without merging**